packages/ui/src/components/StoryShareLinks/StoryShareLinks.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { StoryShareLinks } from '.';
```

Comment:
Better to not re-import from index file and use direct import from the component file